pkg/apiserver/authorizer/authz.go

```diff
@@ -1,5 +1,5 @@
 /*
```

Comment:
Since this is generic (used by both kube-apiserver and federation-apiserver), `pkg/genericapiserver/authorizer` is a better place.

In a separate PR, we should move `pkg/apiserver/authenticator` to `pkg/genericapiserver/authenticator` as well.